tests/test_asset_list.py

```diff
@@ -104,6 +104,8 @@ def cleanup(self):
 
 class AssetListTests(unittest.TestCase):
 
+    maxDiff = 999
```

Comment:
What does this mean?

Reply:
It's a pytest thing that will increase the maximum length of a diff that it will show. I added it to get the output I needed to see at the moment a test was failing and it *could* be removed, but it probably should be kept in because apparently the asset list tests are such that they make errors with very large diffs that need to be printed.